src/witnet/request.js

```diff
@@ -48,6 +49,12 @@ class Request {
 
     return this
   }
+  setCollateral (collateral) {
+    if (collateral < 1000000000){
+      this.data.collateral = collateral || this.data.collateral;
+    }
```

Comment:
Shouldn't this be the other way?
```suggestion
    if (collateral >= 1000000000){
      this.data.collateral = collateral;
    }
```